application/controllers/admin/Tokens.php

```diff
@@ -1438,7 +1441,16 @@ public function email(int $iSurveyId)
                                 $oToken->remindercount++;
                             }
                             $tokenSaveError = "";
-                            if (!$oToken->encryptSave(true)) {
+                            if (
+                                !$oToken->save(
+                                    true,
+                                    array(
```

Comment:
We save only this attribute in the request https://github.com/yiisoft/yii/blob/b031205cf4ec20131639653910a13ca61747da32/framework/db/ar/CActiveRecord.php#L1121

Reply:
Got it